Mathlib/NumberTheory/ZetaFunction.lean

```diff
@@ -6,6 +6,7 @@ Authors: David Loeffler
 import Mathlib.Analysis.SpecialFunctions.Gamma.Beta
 import Mathlib.NumberTheory.ModularForms.JacobiTheta.OneVariable
 import Mathlib.NumberTheory.ZetaValues
+import Mathlib.Analysis.Complex.RemovableSingularity
```

Comment:
Why is this necessary now if it wasn't before?

Reply:
The imports in `JacobiTheta.TwoVariable` changed, so `RemovableSingularity` is no longer transitively imported here and the import needs to be added explicitly.